compiler/x/codegen/UnaryEvaluator.cpp

```diff
@@ -94,6 +94,53 @@ TR::Register *OMR::X86::TreeEvaluator::integerAbsEvaluator(TR::Node *node, TR::C
    return result;
    }
 
+TR::Register*
+OMR::X86::TreeEvaluator::vnegEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::RegisterDependencyConditions *deps = generateRegisterDependencyConditions(0, 1, cg);
+   TR::Node *valueNode = node->getChild(0);
+   TR::Register *resultReg = cg->allocateRegister(TR_FPR);
+   TR::Register *a = cg->evaluate(valueNode);
```

Comment:
At the very least please all it `aReg` to make the code more readable.  Maybe `valueReg` is better because it matches your naming convention for the node where it originated from.